common/playbar.js

```diff
@@ -167,40 +154,22 @@ $(document).ready(function () {
     });
 
     playbarhtml.find("#playbar_toggle").click(function() {
-      console.log("Temp Button");
       updateClip(disable=this.checked);
     });
     playbarhtml.find("#playbar_toggle").trigger('click');
 
     playbarhtml.find("#playbutton").mousedown(function() {
-      console.log("PLAY");
       playbarhtml.find("#playbutton").hide();
       playbarhtml.find("#pausebutton").show();
 
     });
 
     playbarhtml.find("#pausebutton").mousedown(function() {
-      console.log("PAUSE");
       playbarhtml.find("#playbutton").show();
       playbarhtml.find("#pausebutton").hide();
 
     });
 
-    window.addEventListener("keypress", (e) => {
```

Comment:
leave this

Reply:
Going to delete it and track reinstating it in this Zenhub issue: 
https://app.zenhub.com/workspaces/veritas-5b9aa680dfeccd3433cd6f62/issues/nextdroid/potree/24